### PR TITLE
Fix some doc typos

### PR DESCRIPTION
**Pull request by kevinjqiu**
**\- What I did**
Fixing some typos and formatting as I was going through the docs.

**\- How I did it**
N/A

**\- How to verify it**
N/A

**\- Description for the changelog**

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Kevin Jing Qiu kevin@idempotent.ca

### CODE REVIEW

File: docs/userguide/networking/default_network/container-communication.md [hunk 1]
```diff
@@ -72,20 +72,9 @@ Whether two containers can communicate is governed, at the operating system leve
...
-If you choose the most secure setting of `--icc=false`, then how can containers
-communicate in those cases where you _want_ them to provide each other services?
-The answer is the `--link=CONTAINER_NAME_or_ID:ALIAS` option, which was
-mentioned in the previous section because of its effect upon name services.  If
-the Docker daemon is running with both `--icc=false` and `--iptables=true`
-then, when it sees `docker run` invoked with the `--link=` option, the Docker
-server will insert a pair of `iptables` `ACCEPT` rules so that the new
-container can connect to the ports exposed by the other container -- the ports
-that it mentioned in the `EXPOSE` lines of its `Dockerfile`.  
+It is a strategic question whether to leave `--icc=true` or change it to `--icc=false` so that `iptables` will protect other containers -- and the main host -- from having arbitrary ports probed or accessed by a container that gets compromised.
```
Comment (thaJeztah) [inline docs/userguide/networking/default_network/container-communication.md]: can you wrap these lines to 80-chars again? We try to keep the Markdown source wrapped to 80 characters

File: docs/userguide/networking/default_network/ipv6.md [hunk 2]
```diff
@@ -15,12 +15,7 @@ The information in this section explains IPv6 with the Docker default bridge.
 This is a `bridge` network named `bridge` created automatically when you install
 Docker.  
 
-As we are [running out of IPv4
-addresses](http://en.wikipedia.org/wiki/IPv4_address_exhaustion) the IETF has
-standardized an IPv4 successor, [Internet Protocol Version
-6](http://en.wikipedia.org/wiki/IPv6) , in [RFC
-2460](https://www.ietf.org/rfc/rfc2460.txt). Both protocols, IPv4 and IPv6,
-reside on layer 3 of the [OSI model](http://en.wikipedia.org/wiki/OSI_model).
+As we are [running out of IPv4 addresses](http://en.wikipedia.org/wiki/IPv4_address_exhaustion) the IETF has standardized an IPv4 successor, [Internet Protocol Version 6](http://en.wikipedia.org/wiki/IPv6) , in [RFC 2460](https://www.ietf.org/rfc/rfc2460.txt). Both protocols, IPv4 and IPv6, reside on layer 3 of the [OSI model](http://en.wikipedia.org/wiki/OSI_model).
```
Comment (thaJeztah) [inline docs/userguide/networking/default_network/ipv6.md]: can you wrap this line to 80-chars as well?

**Comment by thaJeztah** [comment]
thanks! left some comments, also, can you rebase (and squash) to get rid of the merge commits? you can find some information for that in our contributors guide https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

**Comment by kevinjqiu** [comment]
@thaJeztah Thanks, I've rebased my changes on top of master.

I'm not sure if the CI error is due to changes in this PR though:
```
# Filtered (engine) Summary:

* link error: (in page engine/getstarted/step_one.md) /docker-for-mac/index.md
* link error: (in page engine/getstarted/last_page.md) /docker-for-mac/index.md
* link error: (in page engine/installation/mac.md) /docker-for-mac/index.md
* link error: (in page engine/getstarted/step_one.md) /docker-for-windows/index.md
* link error: (in page engine/installation/windows.md) /docker-for-windows/index.md
* link error: (in page engine/getstarted/last_page.md) /docker-for-windows/index.md

	Found: 820 files
	Found: 6 errors
```

**Comment by thaJeztah** [comment]
Thanks!! Those errors are known, so it's safe to ignore them.

LGTM :+1:

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**